src/analytics.js

```diff
@@ -1,11 +1,30 @@
+import snakecaseKeys from 'snakecase-keys';
 import apiClient from './config/apiClient';
+import { configuration } from './config/environment';
 
+const eventLogApiBaseUrl = `${configuration.LMS_BASE_URL}/event`;
+
+// Sends events to Segment and downstream
 const handleTrackEvents = (eventName, properties) => {
   // Simply forward track events to Segment
   window.analytics.track(eventName, properties);
 };
 
 
+// Sends events to tracking log and downstream
+// TODO: Determine consistent naming for eventName vs eventType and properties v eventData.
+const logEvent = (eventType, eventData) => {
```

Comment:
Do we have an opinion on using this syntax vs below in this context?
```
function logEvent(eventType, eventData) {

}
```

Reply:
I think you do, so I changed it. :)

Reply:
Haha, I don't know if there's a proper way or not

Reply:
The linter should state our proper way, and it is happy with both.  I like the functions more, and you seemed like you might, so I went for it.

Reply:
@douglashall? @davidjoy?